### PR TITLE
Dockerfiles: bump golang to 1.15 ⚓

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Bump golang to 1.15 across images we use

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by afrittoli — COMMENTED** [review]
Thanks, it's good to switch to 1.15.
I hope it won't break anything, even if it might.
Still, Thursday is probably still an ok day to break things :P (after the nightly builds)
/approve
/lgmt

File: scripts/library.sh [hunk 1]
```diff
@@ -339,7 +339,7 @@ function update_licenses() {
    chmod +w $(find ${dst} -type d)
 }
 
-# Run dep-collector to check for forbidden liceses.
+# Check for forbidden liceses.
```
Comment (nikhil-thomas) [inline scripts/library.sh:342]: :heart:

**Pull request merged**